src/Provisioning/ProvisionAssets.ps1

```diff
@@ -76,7 +77,18 @@ Param
 $ErrorActionPreference = "Stop"
 
 # Check the azure module is installed
-Import-Module azure
+if(-not(Get-Module -name "Azure")) 
+{ 
+    if(Get-Module -ListAvailable | Where-Object { $_.name -eq "Azure" }) 
+    { 
+        Import-Module Azure
+    }
+    else
+    {
+        "Windows Azure Powershell has not been installed."
```

Comment:
Fixed the message text.

Regarding running the module check early, I think it can be done with dynamic parameters [as discussed here](https://social.technet.microsoft.com/Forums/windowsserver/en-US/89f13aea-1a21-41ad-b643-9a66edc86dcf/powershell-execute-command-before-param-processing).  I'm not sure if you want to do that or not, as it late-binds the parameters, which removes autocompletion/intellisense of the parameters if one wants to pass everything in one command.   If the intent is to always have the user enter one parameter at a time, then I suppose late-binding is ok.